client/my-sites/site-settings/jetpack-credentials/credentials-form/style.scss

```diff
@@ -34,8 +34,12 @@
 	margin-top: 16px;
 }
 
-.credentials-form__delete-button,
 .credentials-form__cancel-button {
 	float: right;
 	margin-right: 15px;
 }
+
+.credentials-form__delete-button.button.is-borderless {
```

Comment:
Add the `scary` prop to the Button instead of this.

Reply:
Done in 04ca0773ddb2707cf3e1bae7f9feec8c1b03f144